src/components/_proxied-dot-io/common/dev-dot-opt-in/dev-dot-opt-in.module.css

```diff
@@ -0,0 +1,38 @@
+.container {
```

Comment:
Pretty much copy-pasted this from @kendallstrautman's implementation here: https://github.com/hashicorp/learn/pull/4480